### PR TITLE
Cleanup docker files

### PR DESCRIPTION
**Pull request by SchrodingersGat**
Simplification of docker files, conf files, etc.

Closes https://github.com/inventree/InvenTree/issues/1457

Docs PR - https://github.com/inventree/inventree-docs/pull/102

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -50,21 +50,26 @@ services:
...
         restart: unless-stopped
 
     # nginx acts as a reverse proxy
     # static files are served by nginx
     # web requests are redirected to gunicorn
+    # NOTE: You will need to provide a working nginx.conf file!
     nginx:
         container_name: nginx
-        image: inventree/nginx:latest
+        image: nxinx
```
Comment (rcludwick) [inline docker/docker-compose.yml:79]: Typo.  :)

**Comment by rcludwick** [comment]
So with my default config I'm getting something that looks like this: 

```
inventree-web       | Could not connect to database: FATAL:  database "inventree" does not exist
db                  | 2021-04-18 08:50:10.614 UTC [113] FATAL:  database "inventree" does not exist
inventree-worker    | Could not connect to database: FATAL:  database "inventree" does not exist
```

It's easy to fix for me, but it would be nice if it were configured.

**Comment by SchrodingersGat** [comment]
@rcludwick currently (as per the [docker setup guide](https://inventree.readthedocs.io/en/latest/start/docker/#create-database) ) the database is created as a manual step in the setup process.

Are you suggesting that the database should be created automatically? Is this "safe"?

File: docker/docker-compose.yml (hunk 1 above)
Comment (SchrodingersGat) [reply docker/docker-compose.yml]: Thanks, fixed now.

**Comment by rcludwick** [comment]
I pretty sure you just need to run django migrations to create the db for you.  And it's safe.  So django migrations won't destroy any existing DB that exists.

**Comment by rcludwick** [comment]
Looked over your docker changes and they look good to me.

**Comment by SchrodingersGat** [comment]
I do not think that you can auto-create a db (except if you're using the sqlite backend):

https://docs.djangoproject.com/en/3.2/intro/tutorial02/#database-setup

**Comment by rcludwick** [comment]
It looks like you're right.   I use postgres at work for my development database, and I swear I didn't have to, but it's also 3:30am here so I can't really remember right now.  :smile: 

What's weird is that unit tests have no issues creating databases on postgres.... So ...

**Pull request merged**